CONTRIBUTING.md

```diff
@@ -155,3 +159,18 @@ export FIRESTORE_EMULATOR_HOST="localhost:<Firebase emulator PORT>"
 - Every pull request should have associated issue(s) on our [issue tracker](https://github.com/Real-Dev-Squad/website-backend/issues).
 - For any non-trivial fixes and features, unit and integration tests must be added. The PR reviewer should not approve/merge PR(s) that lack these.
 - The PR(s) should be merged only after the CI passes.
+
+## Certain issues you may face while running the tests:
+
+- Firestore emulator is not starting
+- Java version is not above 11
+- When we run yarn test, it runs both the unit and integration tests (in this order). So after the unit tests are done, the java process is not killed automatically and when our integration test run it gives error.
```

Comment:
Do we have any issues created for it in past?
Have you tested it out, when it occurs and how to fix it?

Reply:
Which issue are you refering to?

Reply:
Errors on Line 165 and 167, I have faced myself.
The line 166 is just a prerequisite which is easy to miss, so i mentioned it. @shmbajaj